### ISSUE TITLE
Not detecting windows-curses members

### ISSUE THREAD

**Issue by DRACL10N** [PyCQA/pylint#3685]
Greetings all. I would like to note that I have found one other issue of the same type, but it appears to have had no activity for about 6 months now, so I'm opening this one. 

I installed `windows-curses` and started up VSCode to test it out. I use the Python and Code Runner extensions, and installed `pylint` as my linter as I prefer it.

`Pylint` picks up `curses` as an import and all is well, however for most of `curses`' functions/members it claims that they are non-existent. This is false, as the program runs without a hitch.

### Steps to reproduce
1. `pip install windows-curses`
2. create a python script
3. attempt to use a member such as `curs_set()`

### Current behavior
Raises the error: `Module 'curses' has no 'curs_set' member`.

### Expected behavior
No error raised.

### pylint --version output
```
pylint 2.5.3
astroid 2.4.2
Python 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v1916 64 bit (AMD64)]
```

**Comment by PCManticore** [comment]
`windows-curses` is a C extension module, so pylint cannot analyze it by default since it does not have access to the source code of the library (being in C and all).

There are multiple alternatives here:
* allow pylint to import this module during analysis so it can build an AST out of it, with `--extension-pkg-whitelist=curses`
* Generate all the members yourself with `--generated-members=...`
* Ignore `curses` from analysis with `--ignored-modules=curses`

Hope that helps.
